CONTRIBUTING.md

```diff
@@ -446,6 +449,8 @@ If you get errors running `po2json`, you might be running the Ubuntu package wit
 name, rather than the NodeJS package (they have a different format for the arguments). If
 there is a conflict, you may need to update your `PATH` environment variable or fully qualify
 the executable path (e.g. `/usr/local/bin/po2json` instead of `po2json`).
+If you get a lot of`[null,***]`in `messages.json` just delete all the `null,` 
```

Comment:
- Add space around ``[null,***]``
- full-stop after `null,`
- Add comma after "For example"
- Add full-stop after "is incorrect"

Reply:
Ok，thanks ， i will fix this 4 hours later。

Reply:
I hava fixed this , thanks .